Listenbrainz/UI/Screens/SongDetailView.swift

```diff
@@ -8,38 +8,79 @@
 import SwiftUI
 
 struct SongDetailView: View {
+  @EnvironmentObject var homeViewModel: HomeViewModel
+  @State private var userNameInput: String = ""
+  @State private var listensFetched: Bool = false
+
+  var body: some View {
+    VStack {
+      if !listensFetched {
+        TextField("Enter Username", text: $userNameInput)
```

Comment:
Please remove this from here and just add a username field in the settings page directly

Reply:
done